core/src/main/java/com/ashelkov/wallet/Utils.java

```diff
@@ -31,4 +32,35 @@ public static SecureRandom getSecureRandomInstance() {
         // Default to basic PRNG
         return new SecureRandom();
     }
+
+    /**
+     * Confirms passphrase by prompting user to re-enter and comparing hashes
+     *
+     * @param passphrase passphrase from the first prompt
+     * @return whether or not passphrases match
+     */
+    public static boolean confirmPasswordsMatch(String passphrase) {
```

Comment:
Would you kindly make a new class in `com.ashelkov.wallet.io` (something like `ConsoleUtils`) and move these functions there.